### DISCUSSION TITLE
Create multiple UE

### DISCUSSION THREAD

**Discussion by warlock1302** [herlesupreeth/docker_open5gs#226] [Q&A] [answered]
Hi,
i would like to multiple UE in UERANSIM and start them. 
For eg in UERANSIM we use the command ./build/nr-ue -c config/open5gs-ue.yaml -n 50 for 50 UE's. How can this be achieved inside the docker. appreciate any help.

**Comment by herlesupreeth** [comment, marked as answer]
You can modify here https://github.com/herlesupreeth/docker_open5gs/blob/6fabf5d08eb4f2c767a828854923bfe4b7afc3de/ueransim/ueransim_image_init.sh#L39 and try running it.

NOTE: you need to recompile the UERANSIM UE image for the changes to take effect

**Reply by warlock1302** [reply]
thanks @herlesupreeth .. i have added the new UE in webui. do i also need to update the UE in docker_open5gs/.env file?

**Reply by herlesupreeth** [reply]
I dont think so. UERANSIM just increments the UE IMSI I guess
